### PR TITLE
Website

### PR DESCRIPTION
**Pull request by dominikwilkowski**

### CODE REVIEW

File: website/assets/script.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+function get_os() {
+	const platform = (window.navigator?.userAgentData?.platform || window.navigator.platform).toLowerCase();
+	const macos_platforms = ["macos", "macintosh", "macintel", "macppc", "mac68k"];
+	const windows_platforms = ["win32", "win64", "windows", "wince"];
```
Comment (thewizzy) [inline website/assets/script.js:4]: Is the plan to support targets for windows other than win64

**Review by thewizzy — COMMENTED** [review]
One comment. Looks great!

File: website/assets/script.js (hunk 1 above)
Comment (dominikwilkowski) [reply website/assets/script.js:4]: Maybe? That detection function just checks os. If you're on windows 32 I don't want to show you a download for macOS binaries. No matter if the binaries run on your machine.
Besides would it take much to support windows 32? I assume it's just another target right? For macOS we zip both ARM and intel into one zip. We could do the same for windows. (Eventually I'll make a universal binary for macOS and maybe windows has something like that too)

Comment (thewizzy) [reply website/assets/script.js:4]: 32 and aarch probably not. Windows ce, probably.

Comment (dominikwilkowski) [reply website/assets/script.js:4]: Yeah no need to support windows CE :)

**Review by thewizzy — APPROVED** [review]

**Pull request merged**